src/pkg/bundle/publish.go

```diff
@@ -41,8 +43,38 @@ func (b *Bundle) Publish() error {
 		return err
 	}
 
-	// unarchive bundle into empty tmp dir
-	err = av3.Unarchive(b.cfg.PublishOpts.Source, b.tmp) // todo: awkward to use old version of mholt/archiver
+	// construct a fileHandler that extracts all files from the archive with the same relative pathing
+	fileHandler := func(_ context.Context, file archives.FileInfo) error {
```

Comment:
I see similar functions were added to `utils/utils.go`, should this go there as well?

Reply:
You are right, the `utils.ExtractFile()` helper basically does the exact same thing as this. I will edit that util function and use that instead.